bin/doctrine-migrations.php

```diff
@@ -18,7 +18,7 @@
  */
 
 $autoloadFiles = array(
-    __DIR__ . '/../vendor/autoload.php',
+    __DIR__ . '/../autoload.php',
```

Comment:
This change is wrong. When the project is the root folder, the autoloader will indeed be at ``__DIR__ . '/../vendor/autoload.php'``